src/components/Header.js

```diff
@@ -0,0 +1,18 @@
+import React, {useState} from 'react';
+import { Navbar, HamburguerButton, SearchInput } from './styledTags/Header.style';
+import Toggle from './Toggle';
+
+const Header = ({themeToggler}) => {
+
+    return(
+        <>
+            <Navbar>
+               <HamburguerButton>!!!</HamburguerButton>
+               <SearchInput placeholder="search on wizeline"/>
+               <Toggle onChange={() => themeToggler()}/>
+            </Navbar>
+        </>
```

Comment:
The `Fragment` is not needed here. `Navbar` is enough since it is the only root component